### DISCUSSION TITLE
【ゆる募】イシューテンプレートの使い心地を教えてください

### DISCUSSION THREAD

**Discussion by kagomen** [first-contributions-ja/first-contributions-ja.github.io#44] [【運営】相談・報告]
タイトル通りです。
しばらくテンプレートを使ってみて、

- もっとシンプルな方が良い
- この欄を追加してほしい

など、もし何か思うことがありましたらコメントいただけると幸いです。

個人的には、イシューを書くときは`<!-- コメント -->`があるので特に問題ないのですが、イシューを読む側としては「As-Is To-Beってなんだったっけ❔」って一瞬なるので、

- 「As-Is」→「現状」
- 「To-Be」→「理想の状態」

などと変更しても良いかなあという気もします。
※ As-Is To-Beってなんかかっこいいから、慣れを待つという手もあり……

**Comment by pss-aileen** [comment]
テンプレートは問題の把握や、改善したいこと、作りたいことが整理できるので大変役に立っています🙆‍♀️

英語いいですよね、新しく英語の表現を覚えられるチャンスになりますし、勉強になります。

ただ、As-Is、To-Beは慣れているor知っている人でないとぱっとわからないかもしれません。
（私はAs-Is、知りませんでした...！）

最終的にメンターに誰かが入ること、主に日本人をターゲットにしたリポジトリという点で考えると日本語のほうが親切なのかな？と思いました！

**Reply by kagomen** [reply]
> 最終的にメンターに誰かが入ること、主に日本人をターゲットにしたリポジトリという点で考えると日本語のほうが親切なのかな？と思いました！

そうですよね。ご意見ありがとうございます！🌟

**Comment by kagomen** [comment]
#68 で改良！
